### ISSUE TITLE
all example links in README go to List

### ISSUE THREAD

**Issue by jmhodges** [bvaughn/react-virtualized#520]
In the Examples section of the README, all of the links for Grid, Table, etc. have the List example rendered. The `https://bvaughn.github.io/react-virtualized/?component=Grid` links are redirected to `https://bvaughn.github.io/react-virtualized/?component=Grid#/components/List`. 

That `#/components/List` seems to take precedence.

**Comment by bvaughn** [comment]
The demo site routing has changed since the docs were last updated. Feel free to submit a PR to correct this if you'd like. 😄

**Comment by jmhodges** [comment]
I was just investigating react-virtualized because I'm a newbie. I'm not really in a position to figure that out what is probably a 10 minute deal for someone else. I'm kind of surprised this ticket got closed? Did you make an equivalent one elsewhere or something?

**Comment by bvaughn** [comment]
This project (like many other OSS projects on Github) is maintained by an individual (me) in my free time. Particularly with things like docs- I rely on community to help fix typos, broken links, etc because it's hard for me to manage all of the incoming issues myself and it's pretty simple to update markdown.

I closed this particular issue because I don't think it's important enough to keep open in the issues list. Either I'll fix it or someone else will- maybe you, since you noticed it.

In this case, I already fixed it.
